crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -597,6 +672,46 @@ impl MeshPipeline {
             ))
         }
     }
+
+    pub fn get_view_layout(
+        &self,
+        multisampled: bool,
+        depth_prepass: bool,
+        normal_prepass: bool,
+        motion_vector_prepass: bool,
+        deferred_prepass: bool,
+    ) -> &BindGroupLayout {
+        let index = (multisampled as usize)
+            | (depth_prepass as usize) << 1
```

Comment:
Again relying on operator precedence of bitwise operators, I would want this to be `| ((depth_prepass as usize) << 1)`. Or you could just use the bitflags crate and make a type out of it.